boilerplate/app/screens/DemoComponentsScreen/DemoItem.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { Fragment } from "react"
+import { ListRenderItemInfo, TextStyle, View, ViewStyle } from "react-native"
+import { Text } from "../../components"
+import { Demo } from "./DemoComponentsScreen"
+
+export function DemoItem(props: ListRenderItemInfo<Demo>) {
+  const { item } = props
+
+  return (
+    <View style={$demoItemContainer}>
+      <Text preset="heading" style={$title}>
+        {item.name}
+      </Text>
+      <Text style={$description}>{item.description}</Text>
+
+      {item.useCases.map((i) => (
+        <Fragment key={i.props.name}>{i}</Fragment>
+      ))}
+    </View>
+  )
+}
+
+const $demoItemContainer: ViewStyle = {
+  marginLeft: 24,
```

Comment:
you'll want to use `paddingHorizontal` here to contain the content equally.
![yulolimum-capture-2022-08-12--05-05-35@2x](https://user-images.githubusercontent.com/1775841/184350571-048046d1-9b1a-412a-955a-7de3d18a399d.jpg)